enterprise/server/oidc/oidc.go

```diff
@@ -643,7 +643,7 @@ func (a *OpenIDAuthenticator) authenticateUser(w http.ResponseWriter, r *http.Re
 	// the token below.
 	if ut, err := auth.verifyTokenAndExtractUser(ctx, jwt, true /*=checkExpiry*/); err == nil {
 		claims, err := claims.ClaimsFromSubID(ctx, a.env, ut.GetSubID())
-		if auth.getSlug() != "" {
+		if claims != nil && auth.getSlug() != "" {
```

Comment:
do you need to add one for the check further down?

Reply:
Done